test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json

```diff
@@ -15,19 +15,22 @@
     "Microsoft.Extensions.Logging": "1.0.0-rc2-*",
     "Microsoft.Extensions.Configuration.Json": "1.0.0-rc2-*",
     "xunit": "2.1.0",
-    "FunctionalTestUtils": "1.0.0-rc2-*"
+    "FunctionalTestUtils": "1.0.0-rc2"
```

Comment:
This may have been mentioned in an earlier PR: project-to-project (P2P) references don't need version numbers (because it'll always use whatever version that project is).